test/step-function.test.ts

```diff
@@ -190,9 +236,11 @@ test("let and set", () => {
     a = true;
     a = false;
     a = 0;
+    a = -1;
```

Comment:
do you support other negation?

`-(100)`

`- var`

Reply:
`-var` is not supported. Parens is a good catch.

Reply:
`-(100)` worked but i never handled a ParenthesisExpr, did we auto-remove them?

Reply:
I should also resolve constant computations, e.g. `1 + -1`